docs/automated-testing/getting-started.md

```diff
@@ -0,0 +1,125 @@
+---
+sidebar_label: Getting Started
+---
+
+# Getting Started with Automated Testing
+
+This quick walk-through will show you how to integrate the Regression Games Automated Testing package into your game, 
+and how to use the built-in "Monkey Bot",
+which is a bot that randomly clicks on UI elements in your game,
+recording state along the way. 
+
+## A sample game to start with
+
+If you have a game you want to automate, you can skip this step.
+But if you just want to check the package out and don't have an active game project,
+we recommend using the Boss Room Co-Op Sample Project from Unity.
+You can download it from GitHub: [https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop)
+
+Once downloaded, open the project in Unity.
+Navigate to the **Scenes** folder in the Project window and open the **MainMenu** scene.
+Use this scene for the rest of the tutorial.
+
+![The Unity Project panel showing the "MainMenu" scene in the Boss Room sample.](img/getting-started/bossroom-mainmenu-scene.png)
+
+## Installing the package
+
+Our package is available at [https://github.com/Regression-Games/RegressionGames.Unity.Testing](https://github.com/Regression-Games/RegressionGames.Unity.Testing).
+Import it into your game by clicking **Window > Package Manager** in the Unity menu,
+and then clicking the **+** button in the top left corner of the Package Manager window.
+Select **Add package from git URL** and paste in the following URL:
+
+```
+https://github.com/Regression-Games/RegressionGames.Unity.Testing.git?path=src/gg.regression.unity.testing#v0.0.1
```

Comment:
For testing the doc, I currently recommend using this Git URL instead:

```
https://github.com/Regression-Games/RegressionGames.Unity.Testing.git?path=src/gg.regression.unity.testing#analogrelay/reg-1587
```

When https://github.com/Regression-Games/RegressionGames.Unity.Testing/pull/6 lands, you could also use:

```
https://github.com/Regression-Games/RegressionGames.Unity.Testing.git?path=src/gg.regression.unity.testing
```

To get the latest `main` version

Reply:
Ohhh, I keep forgetting I have this in my global git config 🤣 

```
[url "git@github.com:"]
	insteadOf = https://github.com
```

(It forces any `https://github.com` to become `git@github.com:`, so HTTPS URLs automatically use SSH instead)

Good call, I'll post an updated URL soon.

The doc is still fine though, since when the package goes live, the HTTPS URL will work because the repo will be public.

Reply:
After resolving some issues @abeizer found, please use _this URL_ instead:

```
https://github.com/Regression-Games/RegressionGames.Unity.Testing.git?path=src/gg.regression.unity.testing#ashley/reg-1596
```

This is necessary until https://github.com/Regression-Games/RegressionGames.Unity.Testing/pull/8 lands

Reply:
> I have not http config for git, and even for public repos, it will fail.

We can provide the SSH URL too, but [currently we only provide the HTTPS link in our docs for the existing SDK](https://docs.regression.gg/quickstart#step-2-import-the-regression-games-unity-bots-sdk)

Reply:
Oh my apologies! I confused myself and it was actually the reverse I experienced, duh... we need to use the https one and not the ssh one, THAT was the issue I originally had. So yes we can just keep the HTTPS one